on_node_start.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+
+# https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#file-bash_strict_mode-md
+set -euo pipefail
+
+. /etc/parallelcluster/cfnconfig
+
+echo "Node type: ${cfn_node_type}"
+
+function configure_yum() {
+    cat >>/etc/yum.conf <<EOF
+assumeyes=1
+clean_requirements_on_remove=1
+EOF
+}
+
+function install_fuse_overlayfs() {
+    yum -q install buildah
+
+    if [[ ! -e /dev/fuse ]]; then
+        mknod /dev/fuse -m 0666 c 10 229
+    fi
+
+    pushd "$(mktemp -d)"
+    git clone --depth 1 https://github.com/containers/fuse-overlayfs.git ./overlay
+    pushd ./overlay
+    buildah bud -v "$PWD:/build/fuse-overlayfs" -t fuse-overlayfs -f ./Containerfile.static.ubuntu .
+    cp fuse-overlayfs /usr/bin/
```

Comment:
*non-blocking, query:* Do you not need to `popd` here?

Reply:
I do - and I shouldn't be using `pushd` twice here either. The function isn't actually being used for this script. It is for the ubuntu image on the next MR though, so I'll leave it because this file will be deleted